.gitignore

```diff
@@ -75,4 +75,4 @@ __pycache__
 /util/fmpp*
 
 # Allow to exist but don't include it in the repo
-user_song_list.h
+user_song_list.h
```

Comment:
The change to this file should be reverted too. Which is mostly adding a line at the end, it looks like.

Reply:
```suggestion
user_song_list.h

```